src/main/java/mattern/william/PhoneBook.java

```diff
@@ -0,0 +1,29 @@
+package mattern.william;
+
+import java.util.TreeMap;
+
+/**
+ * Created by williammattern on 2/7/17.
+ */
+public class PhoneBook {
+    TreeMap<String,String> phoneBook = new TreeMap<String, String>();
+
+    public void addEntry(String name, String phoneNumber){
+        phoneBook.put(name, phoneNumber);
+
+    }
+
+    public void deleteEntry(String name){
+        phoneBook.remove(name);
+
+    }
+
+    public String lookup(String name){
+        return phoneBook.get(name);
+    }
+
+    public String getAllNames(){
+        return phoneBook.entrySet().toString();
```

Comment:
You should have two methods here -- one to get all names (just the names) and one to get all entries (names and numbers)